client/src/pages/Country/Country.tsx

```diff
@@ -107,7 +108,32 @@ export const Country: React.FC<CountryProps> = ({ country }) => {
         <br />
       </DropDown>
       <DropDown label="Experimental Metrics" notOpen>
-        <h3>Experimental Metrics</h3>
+      <h3>Experimental Metrics</h3>
+      <h3>Publication Date vs Time to Trend</h3>
+        <LineGraph
+          results={genResults
+            .sort(
+              (a, b) =>
+                new Date(a.pub_date).getTime() - new Date(b.pub_date).getTime()
+            )
+            .map((video) => ({
+              x: video.pub_date,
+              y: video.pub_to_trend,
+            }))}
+        />
+        <div></div>
+        <h3>Publication Date vs Time to Trend</h3>
+        <LineGraph
+          results={genResults
+            .sort(
+              (a, b) =>
+                new Date(a.trend_date).getTime() - new Date(b.trend_date).getTime()
+            )
+            .map((video) => ({
+              x: video.trend_date,
+              y: video.comment_count,
+            }))}
+        />
```

Comment:
Graphs are under 'Experimental Metrics', but don't use `expMetrics`. Might be possible to do:
`genResults.concat(expResults)`, or move these up into the 'General Metrics'

Reply:
Moved graphs under general metrics for now. Might want to discuss tomorrow if we want the graphs side by side, in a drop down menu, still stacked, etc.